### ISSUE TITLE
Add test endpoint for consent

### ISSUE THREAD

**Issue by phavekes** [OpenConext/OpenConext-engineblock#1761]
[This issue is imported from pivotal - Originaly created at Feb 7, 2019 by bstrooband](https://www.pivotaltracker.com/story/show/163792703)

Due to the fact that testing the wayf and consent pages take a fair amount of time, maybe we could add an endpoint to easily test these page with minimal effort.

**Edit feb 27**
The WAYF test endpoint is created in: #159650389. The scope of this ticket can be narrowed to the consent screen.

**Comment by phavekes** [comment]
Note that the debug endpoint already directly triggers a wayf. For consent, this is indeed more involved right now and might be nice to trigger it more easily. (Thijs Kinkhorst - Feb 13, 2019)

**Comment by phavekes** [comment]
This has been started or even done? (Thijs Kinkhorst - Feb 26, 2019)

**Comment by phavekes** [comment]
@thijskh Yes this is started in part, the WAYF test endpoint is created/tested in #159650389.

I suggest we update this story, and focus it on the Consent screen, which we also want to test, and put it back in the backlog/icebox. (Michiel Kodde - Feb 27, 2019)

**Comment by phavekes** [comment]
@thijskh this is implemented in 6.2 in order to easily test the upgrading of NPM dependencies. This is currently only active in dev and test mode and is only for a single relatively default situation to test some visual regressions. We could now easily add some more fine grained tests if required. I think it could receive a bit more TLC in the long run but will be sufficient for now.

`https://github.com/OpenConext/OpenConext-engineblock/pull/840`
`https://engine.vm.openconext.org/functional-testing/consent` (bstrooband - Apr 1, 2020)

**Comment by baszoetekouw** [comment]
Seems to be done.
